geode-lucene/src/main/java/org/apache/geode/cache/lucene/internal/distributed/WaitUntilFlushedFunction.java

```diff
@@ -85,8 +72,10 @@ public void execute(FunctionContext context) {
       }
 
     } else {
-      throw new IllegalArgumentException(
+      IllegalStateException illegalStateException = new IllegalStateException(
           "The AEQ does not exist for the index " + indexName + " region " + region.getFullPath());
+      logger.error(illegalStateException.getMessage());
```

Comment:
I still think we should just throw the exception here.  The executing side will either get a true or false anyways and the exception never actually gets propagated to the user.